client/src/components/SearchResultList/SearchResultList.js

```diff
@@ -0,0 +1,5 @@
+import styled from "@emotion/styled";
+
+const SearchResultList = styled.nav``;
```

Comment:
Do you really need this styled component if you don't add any styles to it? Can't you just use a normal HTML-nav element?

Reply:
Yes, you are right! I did it in case later I added some styling, but I could simple use a nav :) thanks!